src/main/java/com/google/api/codegen/viewmodel/InitCodeView.java

```diff
@@ -33,6 +33,9 @@
    */
   public abstract String apiFileName();
 
+  /** The file name of the index file. */
+  public abstract String indexFileImportName();
```

Comment:
I think 'versionIndexFileName' is better since if we use 'ImportName', it feels that this should be part of ImportSectionView